css/style.css

```diff
@@ -1,25 +1,25 @@
-body{
-	background: #EBEBEB;
-}
-header,
-main,
-footer
- {
-    width: 1110px;
+html body {
+	background: #dfdfdf;
+	width: 1110px;
     margin: 0 auto;
-	
+	padding: 0;
+
 }
 .topbar{
 	display: flex;
 	align-items: center;
 	justify-content: space-between;
 }
+.col-7.topbar_list {
+    padding: 0;
+}
 .topbar_list{
 	display: flex;
 	flex-direction: row;
 	align-items: center;
 	gap: 28px;
 	font: 800 16px Roboto, sans-serif;
+	justify-content: space-between;
 }
 .topbar_list_little{
```

Comment:
Please use dash (-) for class names with more than one word instead of underscore ( _ )

Пожалуйста используй тире (-) для имен класса где больше чем одно слово вместо нижнего подчеркивания ( _ )